lib/SIL/SILUndef.cpp

```diff
@@ -16,6 +16,15 @@
 using namespace swift;
 
 static ValueOwnershipKind getOwnershipKindForUndef(SILType type, const SILFunction &f) {
+  if (type.isAddress()) {
+    // If we have an address only type and we are supposed to use
+    // lowered addresses, return Owned. Otherwise addresses are any.
+    if (type.isAddressOnly(f) && f.getConventions().useLoweredAddresses()) {
+      return ValueOwnershipKind::Owned;
```

Comment:
Let's not do anything special for SIL opaque values here. Let's just treat address types as trivial (i.e. Any)

Reply:
@atrick https://github.com/apple/swift/pull/25979